editors/vim/lua/lsp.lua

```diff
@@ -0,0 +1,21 @@
+local client = nil
+
+local function start()
+  if client == nil then
+    local script_dir = debug.getinfo(1, "S").source:match('@?(.*/)')
+    local plugin_dir = vim.fn.resolve(script_dir .. '../')
+    local ls_path = vim.fn.resolve(plugin_dir .. '/../vscode/out/server/src/server.js')
```

Comment:
does this mean you can only start nvim from a particular directory?

Reply:
nope. `script_dir` is set so it always points to `<jakt symlink by package manager>/editors/vim/lua/`.
I use `resolve` to resolve the symlink and make the plugin dir point to `<path/to/jakt>/editors/vim`, so I can access `editors/vscode/out/...` correctly. I'l add comments to avoid confusion.